packages/agent-kit/src/boards/specialist.ts

```diff
@@ -74,8 +79,45 @@ You are very creative and you pride yourself in adding interesting twists and un
 const sampleTask: LlmContent = contextFromText(`
 Write an outline for a novel, following the provided specs.
 `);
+const context = input({
+  title: "Context in",
+  description: "Incoming conversation context",
+  type: annotate(array("string"), {
+    behavior: ["llm-content"],
+  }),
+  examples: [[JSON.stringify(sampleContext, null, 2)]]
+});
+
+const persona = input({
+  title: "Persona",
+  description: "Describe the worker's skills, capabilities, mindset, and thinking process",
+  type: annotate(object({}), {
+    behavior: ["llm-content", "config"],
+  }),
+  examples: [[JSON.stringify(samplePersona, null, 2)]]
+});
+
+const task = input({
+  title: "Task",
+  description: "(Optional) Provide a specific task with clear instructions for the worker to complete using the conversation context",
+  type: annotate(object({}), {
+    behavior: ["llm-content", "config"],
+  }),
+  default: {},
+  examples: [[JSON.stringify(sampleTask, null, 2)]],
+});
+
+const tools = input({
+  title: "Tools",
+  description: "(Optional) Add tools to this list for the worker to use when needed",
+  type: annotate(array("string"), {
+    behavior: ["board", "config"],
+  }),
+  default: [],
+  examples: [[JSON.stringify(sampleTask, null, 2)]],
+});
 
-const specialist = await board(({ in: context, persona, task, tools }) => {
+const specialist = await oldBoard(({ in: context, persona, task, tools }) => {
```

Comment:
It shouldn't be possible to mix the old and new APIs within a single board. But I assume this is just a partial conversion for the draft PR yeah?